modules/tensor_mechanics/test/tests/action/tests

```diff
@@ -87,29 +87,23 @@
   [./eigenstrain_two_blocks_names]
     type = RunApp
     input = 'action_multi_eigenstrain.i'
-    expect_out = 'block1: eigenstrain1
-block2: eigenstrain2'
-    match_literal = true
+    expect_out = 'block1: eigenstrain1\n.*\nblock2: eigenstrain2'
```

Comment:
All of those should end with a `\n` to guard against matching an output that has more eigenstains appended to it.